save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt

```diff
@@ -42,4 +43,18 @@ class ExecutionController(private val executionService: ExecutionService) {
             executionService.getExecutionDto(executionId)?.let {
                 ResponseEntity.status(HttpStatus.OK).body(it)
             } ?: ResponseEntity.status(HttpStatus.NOT_FOUND).build()
+
+    /**
+     * @param name
+     * @param owner
+     * @return list of execution dtos
+     */
+    @GetMapping("/executionDtoByNameAndOwner")
```

Comment:
```suggestion
    @GetMapping("/executionDtoList")
```
I don't think specifying parameters in method name is good here, it's more important that we get several objects, not just one